service/src/main/java/uk/gov/hmcts/reform/fpl/service/GeneratedOrderService.java

```diff
@@ -132,7 +132,7 @@ public OrderTypeAndDocument buildOrderTypeAndDocument(OrderTypeAndDocument typeA
                     .put("epoType", caseData.getEpoType())
                     .put("includePhrase", caseData.getEpoPhrase().getIncludePhrase())
                     .put("removalAddress", getFormattedRemovalAddress(caseData))
-                    .put("childrenCount", caseData.getChildren1().size())
+                    .put("childrenCount", caseData.getChildren1() != null ? caseData.getChildren1().size() : 0)
```

Comment:
Is this used in the document, thought we discussed to remove it? Or was it just the order doc where we removed it?

Reply:
Yeah it's used in my EPO template. Joe's children logic does a bit more formatting than mine does. I just use the childrenCount to display either 'Child' or 'Children' in a few places.